pages/index.jsx

```diff
@@ -1,6 +1,11 @@
 import RecipeList from "../components/RecipesList/RecipeList";
+import EnvError from "./error";
 
 const Home = () => {
+  if (process.env.NEXT_PUBLIC_ENV_VAR) {
```

Comment:
@shaniquah Did you perhaps get this line of code online?

Reply:
@Cbazil I did while looking for new ways to resolve my issue with the same functionality given the new code changes. I was unable to make the code work on my side with using only `if(process.env)` as I kept getting an error saying that `process` is undefined and had to find ways around it. i hope there aren't any problems with this line of code being used?